<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFtpDemoClass</class>
 <widget class="QWidget" name="QFtpDemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QFtpDemo</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="upBtn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>upload</string>
   </property>
  </widget>
  <widget class="QPushButton" name="downBtn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>download</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qftpdemo.qrc"/>
 </resources>
 <connections/>
</ui>
